makefile

```diff
@@ -12,10 +12,11 @@ VERSION=v$(TODAY)
 all: build stage clean analyze
 # analyze: runs more than just this file; that goal creates multiple files
 analyze: output/medgen_terms_mapping_status.tsv
-build: $(PRODUCTS) medgen.sssom.tsv
+build: $(PRODUCTS) medgen.sssom.tsv ROBOT_addMedGen_fromIngest.tsv
 stage: $(patsubst %, stage-%, $(PRODUCTS))
 	mv medgen.obo output/release/
 	mv medgen.sssom.tsv output/release/
+	mv ROBOT_addMedGen_fromIngest.tsv output/release/
```

Comment:
```suggestion
	mv medgen-mappings.robot.tsv output/release/
```

Reply:
I implemented this suggestion as amended commit.